localization/english/event_localization/nickname/wc_nickname_events_l_english.yml

```diff
@@ -0,0 +1,17 @@
+﻿l_english:
+ WCNIC.0001.t:0 "The Oldest One"
```

Comment:
I'd get rid of the "One" at the end. "The Oldest" would be better imho.

Reply:
done in 8ac35fd1